app/controllers/user.js

```diff
@@ -44,6 +44,46 @@ exports.create = (req, res, next) => {
   
 };
 
+exports.signin = (req, res, next) => {
+
+  let input = emptyToNull(req.body);
+
+  if(!input.email || !input.password){
```

Comment:
To check params presence or validation you can use some package to help you and avoid some trouble, for example I know the followings:
https://github.com/ssowonny/strong-params
https://github.com/ctavan/express-validator